src/server.js

```diff
@@ -86,16 +97,23 @@ const main = async () => {
    * @apiParam {Date} properties.timestamp Timestamp of the event
    * @apiSuccess (Success 200) -
    */
-  app.post('/api/event', promiseHandler(withAuth(async req => {
-    const { event, properties } = req.body
+  app.post('/api/event', promiseHandler(withAuth(trackEvent)))
 
-    validateInput(req.body, Joi.object({ event: eventSchema, properties: propertiesSchema }))
-
-    log(req.body)
-    token && sendToMixpanel(token, event, properties)
-
-    return new Response(200)
-  })))
+  /**
+   * @api {post} /api/event/unregistered Log a user event
+   * @apiDescription Records the event for an unregistered user to a log and forwards it to mixpanel
+   * @apiName event
+   * @apiVersion 1.0.0
+   * @apiGroup Events
+   * @apiParam {String} event Name of the event
+   * @apiParam {Object} properties Properties associated with this event. The below fields are required. Additional application defined fields can also be used
+   * @apiParam {String} properties.userId The anonymized ID of the user
+   * @apiParam {String} properties.appId The application
+   * @apiParam {String} properties.appPath The navigational path in the application the user is on, with identifying parameters removed
+   * @apiParam {Date} properties.timestamp Timestamp of the event
+   * @apiSuccess (Success 200) -
+   */
+  app.post('/api/event/unregistered', promiseHandler(trackEvent))
```

Comment:
TOL: Do we need a separate endpoint for unregistered user events or should we just detect whether or not there's an `authorization` header? It looks to me like the API, including all of the expected required parameters, is the same for both cases.

Also, historically in the FireCloud API, we put `/api/` only in front of all of the endpoints that required auth. I think that was for the purpose of configuring the LDAP module in apache httpd (or something like that) so we may not need to do that anymore. If we want to keep that same pattern, that would be a reason for a separate endpoint. However, I don't think we have a strong reason to do that so I still vote for merging them if there isn't any other reason not to.